website/docs/developer-section/quickstart/create-verifiable-document-issuer.md

```diff
@@ -134,7 +134,7 @@ const App = () => {
 export default App;
 ```
 
-When you reload the app, the MetaMask extension should prompt you for your password and ask for permission to allow the site access to your accounts.
+When you reload the app, the MetaMask extension should prompt you for your password and ask for permission, so that the site can access your accounts.
```

Comment:
Can we revert this rephrase? In the context of permissions we should still say "allow"

Reply:
reverted